main.js

```diff
@@ -1,9 +1,17 @@
 'use strict';
 
-const { createServer } = require('./src/createServer');
+const express = require('express');
+const app = express();
 
-createServer()
-  .listen(3000, () => {
-    // eslint-disable-next-line no-console
-    console.log('Server is running on localhost:3000');
-  });
+app.use(express.json());
+
+const usersRouter = require('./src/routers/usersRouter');
+const expensesRouter = require('./src/routers/expensesRouter');
+
+app.use('/users', usersRouter);
+app.use('/expenses', expensesRouter);
+
+app.listen(process.env.PORT || 3000, () => {
```

Comment:
You duplicate this code in line 16 too so its better way to store this value in variable like const PORT = process.env.PORT || 3000